pkg/csi_driver/controller_test.go

```diff
@@ -449,38 +449,39 @@ func TestCreateVolume(t *testing.T) {
 		features        *GCFSDriverFeatureOptions
 		expectedOptions []*file.NfsExportOptions
 	}{
-		{
-			name: "valid defaults",
-			req: &csi.CreateVolumeRequest{
-				Name: testCSIVolume,
-				VolumeCapabilities: []*csi.VolumeCapability{
-					{
-						AccessType: &csi.VolumeCapability_Mount{
-							Mount: &csi.VolumeCapability_MountVolume{},
-						},
-						AccessMode: &csi.VolumeCapability_AccessMode{
-							Mode: csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER,
-						},
-					},
-				},
-				Parameters: map[string]string{
-					"tier":     zonalTier,
-					"protocol": v4_1FileProtocol,
-				},
-			},
-			resp: &csi.CreateVolumeResponse{
-				Volume: &csi.Volume{
-					CapacityBytes: 1 * util.Tb,
-					VolumeId:      testVolumeID,
-					VolumeContext: map[string]string{
-						attrIP:           testIP,
-						attrVolume:       newInstanceVolume,
-						attrFileProtocol: v4_1FileProtocol,
-					},
-				},
-			},
-			features: features,
-		},
+		// TODO(riteshghorse): Add tests once NFSv4 is supported
```

Comment:
I'd rather remove the commented-out code and just leave the TODO.

Reply:
done